backend/Exceptions/File/NoNameFileException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Exceptions\File;
+
+class NoNameFileException extends FileException
```

Comment:
I think that in all exception classes you should add http codes e.g. 422, 404 etc.
U can choose which type of code will be okey - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
It's static site and they won't be thrown at site's runtime. However, I think it's good idea to include them for simplified information and possible further development.